test/SourceKit/DocumentStructure/rdar47426948.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %sourcekitd-test -req=structure %s -- %s | %FileCheck %s
+
+class C {
+  @IBAction init(foo: Void) {}
```

Comment:
I think the bug also happens with `@objc`; you don't need `@IBAction`

Reply:
It doesn't happen with `@objc` because ObjC selector name is only resolved for `@IBAction`. https://github.com/apple/swift/blob/ff0d764ec922a5d69c2466499120fea0afe1c854/tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp#L1275-L1277